### ISSUE TITLE
Get the same result with SimpleRNN

### ISSUE THREAD

**Issue by zaoyehuo** [braingineer/ikelos#5]
Hello @braingineer 
Thanks for your code of the CWRNN.I am going to implement CWRNN for my work so I tried it with keras(version 1.2.2)and I found something strange.
''''''''''''''''''''''
model = Sequential()
model.add(SimpleRNN(100,
                    batch_input_shape=(batch_size,tsteps,1),
                    return_sequences=True,
                    stateful=False))   
model.add(SimpleRNN(100,
                    return_sequences=False,
                    stateful=False))                           
model.add(Dense(1))
model.compile(loss='mse', optimizer=opt)
'''''''''''''''''''''''
model = Sequential()
model.add(ClockworkRNN(100,period_spec=[1,2,4,8,16],
                    batch_input_shape=(batch_size,tsteps,1),
                    return_sequences=True,
                    stateful=False))   
model.add(ClockworkRNN(100,period_spec=[1,2,4,8,16],
                    return_sequences=False,
                    stateful=False))                           
model.add(Dense(1))
model.compile(loss='mse', optimizer=opt)
''''''''''''''''''''''
I got the same loss with these 2 codes.I don't know what's wrong with my code.
Thank you!
